packages/frontend-core/src/api/user.js

```diff
@@ -141,20 +141,18 @@ export const buildUserEndpoints = API => ({
 
   /**
    * Invites multiple users to the current tenant.
-   * @param email An array of email addresses
-   * @param builder whether the user should be a global builder
-   * @param admin whether the user should be a global admin
+   * @param users An array of users to invite
    */
-  inviteUsers: async ({ emails, builder, admin }) => {
+  inviteUsers: async users => {
     return await API.post({
       url: "/api/global/users/inviteMultiple",
```

Comment:
I didn't actually notice this in the original PR for groups, but this URL isn't great, camel casing in resources is always a bit weird since URLs aren't case sensitive, ideally I think this should probably be `/api/global/users/multi/invite` - incase there are other multi actions we want to perform around groups of users in the future.